src/docs/spec-formats-tsv.md

```diff
@@ -208,6 +230,13 @@ When writing the metadata block, a canonical SSSOM/TSV writer:
 * MUST NOT include in the CURIE map any prefix name that is not used anywhere in the set;
 * MUST sort the prefix names in the CURIE map in lexicographical order.
 
+In addition, if [extension slots](spec-model.md#non-standard-slots) are supported, the writer:
+
+* MUST write any extension slot in the mapping set _after_ the standard slots;
+* MUST sort the extension slots lexicographically on the `property` of their corresponding extension definitions;
+* MUST sort extension definitions on their `property` value;
+* MUST not include an extension definition if the corresponding extension is not used anywhere in the set.
```

Comment:
NON-ACTIONABLE COMMENT: I am a bit ambivalent about this one, similar to the question of rendering empty columns, as there is a use case for serialising mapping sets that are "work in progress", i.e. you pipe a SSSOM TSV through one process (say "filter) and then publish it so it can be edited my a human workflow. I just want to put this thought out there, not block this specific restriction as the user can always use the flag for not canonicalising if they have this specific need.

Reply:
Resolving this.